test/inspector/test-inspector-port-zero-cluster.js

```diff
@@ -6,27 +6,29 @@ const assert = require('assert');
 const cluster = require('cluster');
 
 if (cluster.isMaster) {
+  cluster.on('online', common.mustCall((worker) => worker.send('debugPort'), 3));
+  cluster.on('error', (err) => assert.fail(err));
   const ports = [];
-  for (const worker of [cluster.fork(),
-                        cluster.fork(),
-                        cluster.fork()]) {
-    worker.on('message', common.mustCall((message) => {
-      ports.push(message.debugPort);
-      worker.kill();
-    }));
-    worker.send('debugPort');
+  function serialFork() {
+    return new Promise((res) => {
+      const worker = cluster.fork();
+      worker.on('message', common.mustCall((message) => {
+        worker.kill();
+        ports.push(message.debugPort);
+        res();
+      }));
+    });
   }
-  process.on('exit', () => {
-    ports.sort();
+  serialFork().then(serialFork).then(serialFork).then(common.mustCall(() => {
     assert.strictEqual(ports.length, 3);
     assert(ports.every((port) => port > 0));
     assert(ports.every((port) => port < 65536));
     assert.strictEqual(ports[0] + 1, ports[1]);  // Ports should be consecutive.
     assert.strictEqual(ports[1] + 1, ports[2]);
-  });
+  })).catch((err) => console.error(err));
 } else {
-  process.on('message', (message) => {
+  process.on('message', common.mustCall((message) => {
```

Comment:
`common.mustCall()` failing in a worker won't do anything, so I wouldn't include it here because it gives the reader a false sense that it's an effective check. (You can test this by passing `common.mustCall(fn, 1000)` or something like that.)